index.js

```diff
@@ -1,10 +1,32 @@
-const equals = require('deep-equal');
+// deep equals
+const isObject = (obj) => typeof obj === "object" && obj !== null;
+
+// TODO does this work on arrays?
```

Comment:
I think yes! Write a test ...

Reply:
I was putting it of because of because then I would have needed another file and I was worried that would add more bundle size. but I guess I can add it as part of a `.equals` test